.github/workflows/build.yml

```diff
@@ -0,0 +1,44 @@
+name: Build Platform UI
+concurrency: 
+    group: ${{ github.workflow }}-${{ github.ref }}
+    cancel-in-progress: true
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0 # required for jgit timestamp provider to work
+      uses: actions/setup-java@v2
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+        cache: maven
+    - name: Build with Maven
+      uses: GabrielBB/xvfb-action@v1
+      with:
+       run: >- 
+        mvn
+        --batch-mode
+        -Pbuild-individual-bundles
+        -Pbree-libs
+        -Papi-check
+        -Dcompare-version-with-baselines.skip=false
+        -Dproject.build.sourceEncoding=UTF-8
+        -Dorg.slf4j.simpleLogger.showDateTime=true 
+        -Dorg.slf4j.simpleLogger.dateTimeFormat=HH:mm:ss.SSS
+        -DtrimStackTrace=false
```

Comment:
You might want to add property t not fail on failing tests and using multiple threads.

Reply:
I tried to use the same settings as Jenkins build file. We should change both if we want to but I would avoid doing this for the intial contribution.

Reply:
do not failed with  test ist a requirement for the unitest action to work.